helm/cas-ciip-portal/values-wksv3k-test.yaml

```
@@ -4,14 +4,22 @@ route:
   insecure: false # cert has been issued
 
 env:
-  smtpConnectionString: smtp://example.com
+  smtpConnectionString: smtp://cas-mailhog:1025
```

```suggestion
  smtpConnectionString: smtp://cas-ciip-portal-mailhog:1025
```
This needs to match the mailhog service name